core/Pos/Aeson/Core.hs

```diff
@@ -0,0 +1,62 @@
+-- | Aeson instances for core
+
+module Pos.Aeson.Core
+       (
+       ) where
+
+import           Universum
+
+import           Data.Aeson       (ToJSON (toJSON), FromJSON (..))
+import           Data.Aeson.TH    (defaultOptions, deriveToJSON, deriveJSON, deriveFromJSON)
+import           Data.Time.Units  (Millisecond, Microsecond)
+import           Serokell.Util.Base64  (JsonByteString (..))
```

Comment:
Why Base64? I believe we agreed to use Base16 :/

Reply:
it's used for bytesting b/c there is convenient `JsonByteString` for base64 and I decided it doesn't matter we use `base16` or `base64` but using this wrapper is less error-prone